lib/teleplug.ex

```diff
@@ -6,23 +6,34 @@ defmodule Teleplug do
   alias Plug.Conn
 
   require Logger
-  require OpenTelemetry.SemanticConventions.Trace, as: Conventions
+  require OpenTelemetry.SemConv.ClientAttributes, as: ClientAttributes
```

Comment:
Are we sure we need a require and not an alias here?

Reply:
Not really honestly, I don't know why they were implemented as require originally but I have a feeling they should be `alias`... let me change that

Reply:
I think `Trace` has some kind of magic inside of it. But i don't think this applies to the other `SemConv` modules

Reply:
Agreed